charger/ocpp.go

```diff
@@ -204,6 +215,13 @@ func NewOCPP(id string, connector int, idtag string,
 							c.idtag = *opt.Value
 							c.log.DEBUG.Printf("overriding default `idTag` with Alfen-specific value: %s", c.idtag)
 						}
+
+					case ocpp.KeyChargingScheduleAllowedChargingRateUnit:
```

Comment:
In der Config ist "A" schon Default. Warum hier überschreiben?

Reply:
Ich bin eher so der Fan vom Nochmalhinschreiben. Kann ich aber bereinigen.